src/Vault.sol

```diff
@@ -570,28 +621,28 @@ contract Vault is ERC4626, ERC20Permit, ILiquidationSource, Ownable {
   function _availableBalanceOf(address _token) internal view returns (uint256) {
```

Comment:
I feel like we should rename this function `_liquidatableBalanceOf` because it's too generic otherwise.  It gets confusing!

We can tackle this later.  Just noting it